### ISSUE TITLE
Use Brand with Script Link component on services with variants

### ISSUE THREAD

**Issue by j-pendlebury** [bbc/simorgh#4009]
⚠️ This should be done at the same time as the Navigation is integrated, as we moved the Skip to content link from the Navigation to the Brand. ⚠️

**Is your feature request related to a problem? Please describe.**
Import the `Script Link` component into Simorgh and use on:
- Serbian
- UKChina
- Zhongwen

**Describe the solution you'd like**
Use the alpha Brand component which includes the Script Link -https://github.com/bbc/psammead/pull/2474

The script link contains an `onClick` prop, which can be passed an event handler that sets the variant preference cookie when the user interacts with the link.

Add translations for variants:

Site | Variant | Translation
-- | -- | --
Serbian | bbc.com/serbian/lat | Lat
Serbian | bbc.com/serbian/cyr | Ћир
UK China | bbc.com/ukchina/simp | 简
UK China | bbc.com/ukchina/trad | 繁
Zhongwen | bbc.com/zhongwen/trad | 简
Zhongwen | bbc.com/zhongwen/simp | 繁

**Describe alternatives you've considered**

**Testing notes**
Will need functionality testing
Will need cypress tests
Will need cross-browser testing
Will need some a11y testing

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
WSFRONTPGS-100

**Comment by AlistairGempf** [comment]
Display site not currently on in component, eg. display "Lat" on Cyrillic Serbian site.

**Comment by AlistairGempf** [comment]
Notify core pod when picking this ticket up.

**Comment by AlistairGempf** [comment]
Talk @jamesbrumpton about cypress tests re: what is to be e2e and what is unit tests.

**Comment by AlistairGempf** [comment]
Blocked on bbc/psammead#2256

**Comment by DenisHdz** [comment]
The alpha component is already in Psammead, so it can be integrate it locally in Simorgh to be able to do the [swarm](https://github.com/bbc/psammead/issues/2276)

**Comment by Bopchy** [comment]
Blocked on https://github.com/bbc/psammead/issues/2467

**Comment by Bopchy** [comment]
Blocked on https://github.com/bbc/psammead/issues/2454

**Comment by Bopchy** [comment]
This work has been done [here](https://github.com/bbc/simorgh/pull/4948) and merged into the [Navigation feature branch](https://github.com/bbc/simorgh/pull/4833) which is not ready to go live until all of the work is complete.
